lib/utils.pm

```diff
@@ -954,7 +955,13 @@ sub ensure_serialdev_permissions {
     # ownership has effect immediately, group change is for effect after
     # reboot an alternative https://superuser.com/a/609141/327890 would need
     # handling of optional sudo password prompt within the exec
-    assert_script_run "chown $testapi::username /dev/$testapi::serialdev && gpasswd -a $testapi::username \$(stat -c %G /dev/$testapi::serialdev)";
+    if (check_var('VERSION', 'SLES-11')) {
```

Comment:
Same as above, not safe to check 'SLES-11' since we never define such version. sle_version_at_least('12') is better if implemented.